lib/HeroBonus.h

```diff
@@ -362,6 +366,8 @@ struct DLL_LINKAGE Bonus : public std::enable_shared_from_this<Bonus>
 		h & effectRange;
 		h & limiter;
 		h & propagator;
+		if(version >= 781)
+			h & updater;

```

Comment:
We come up with idea that it's could be good for serialization code to always have brackets, even for single line.

https://wiki.vcmi.eu/Coding_guidelines#Serialization